apps/src/blockly/customBlocks/googleBlockly/behaviorBlocks.js

```diff
@@ -184,26 +184,13 @@ GoogleBlockly.Extensions.registerMutator(
  * @param {WorkspaceSvg} workspace The workspace containing procedures.
  * @returns an array of XML block elements
  */
-export function flyoutCategory(workspace, functionEditorOpen = false) {
+export function flyoutCategory(workspace) {
   const blockList = [];
 
-  const behaviorDefinitionBlock = {
```

Comment:
Can you clarify why this was removed? I see it's not currently used, but I don't remember when we stopped needing it.

Reply:
It was used to define an empty c-shaped behavior definition block that was placed in the toolbox when the modal editor was disabled. We do this for functions, but not behaviors because behaviors are always hidden from the main workspace. I originally added this back in the summer as a way of testing our earliest behavior blocks.